assets/scripts/datalayer/common.js

```diff
@@ -425,6 +427,32 @@ document.shopAnalytics = {
     removeProductsFromCart($cart_items);
   }
 
+  /**
+   * Populates tracking data for the clicked nav item.
+   */
+  function onNavItemClick(e) {
+    var nav = null;
+    var context = '';
+
+    if (nav = this.closest('nav[aria-label]')) {
+      context = nav.getAttribute('aria-label');
+    }
+    else if (nav = this.closest('nav[id]')) {
+      context = nav.getAttribute('id');
+    }
+    else {
+      context = 'unknown';
+    }
+
+    var event_data = {
+      'event': 'UniversalEvent',
+      'eventCategory': 'User Interaction | ' + context.toLowerCase() + ' menu',
+      'eventAction': 'Click',
+      'eventLabel': this.innerText.toLowerCase(),
```

Comment:
@daschmidt94, @LucaPipolo as menu items could be written inconsistently, e.g. `KITCHENS`, `my KITCHEN`, `My Kitchen` I decided to transform the text to all lowercase. The same applies for the aria label or ID attribute.

Also what value should be sent, if neither an aria-label or an ID attribute is set? Should we then prevent sending data at all?